app/main.py

```diff
@@ -21,10 +23,12 @@ def __init__(self, model=None):
     def predict(self, input):
         if 'image' not in input:
             raise KeyError("No key named 'image' in input.")
-        try:
+        if is_base64_str(input['image']):
```

Comment:
Change it to have 2 separate inputs:
- `input_url` or `image_string`

Have validation check to see if it passes the XOR (ie: one must exist but not both).